app/js/controllers.js

```diff
@@ -1,5 +1,7 @@
-var conceptDirControllers = angular.module('conceptDirControllers', []);
+var conceptDictControllers = angular.module('conceptDictControllers', []);
+
+conceptDictControllers.controller('ClassesListCtrl', ['$scope', 'Classes', function($scope, Classes) {
+	  $scope.classObj = Classes.query();	 
```

Comment:
When we change it to in services.js line 8:
query: {method:'GET', isArray:true}

we get an error: "Error in resource configuration for action `query`. Expected response to contain an array but got an object"